src/auto-evo/AutoEvoRun.cs

```diff
@@ -242,7 +242,7 @@ public bool IsFinished(bool autoStart = true)
     /// <remarks>
     ///   <para>
     ///     This has to be called after this run is finished.
-    ///     <see cref="CalculateFinalExternalEffectSizes"/> must be called first,
+    ///     <see cref="CalculateAndApplyFinalExternalEffectSizes"/> must be called first,
     ///     that should be called even before generating the result summaries to make sure they are accurate.
     ///   </para>
     /// </remarks>
```

Comment:
Just one small comment, should `ApplyAllResultsAndEffects` now be renamed to `ApplyAllResults`? As it no longer handles applying the external effects.

Reply:
The summary on this method is `Applies things added by addExternalPopulationEffect and auto-evo results` but I think it should now be `Applies computed auto-evo results to the world` as with the name this method is no longer responsible for handling the external effects instead the computed results need to be the final numbers to apply.

Reply:
Ah, okay, that makes sense.